Week1/credit/credit.c

```diff
@@ -0,0 +1,85 @@
+#include <cs50.h>
+#include <stdio.h>
+
+int main(void)
+{
+    // input creditcard number
+    long n = get_long("Creditcardnumber: ");
+
+    // Count lenght of Creditcardnumber
+    int i = 0;
+    long cc = n;
+    while (cc > 0)
+    {
+        cc = cc / 10;
+        i++;
+    }
+
+    // Check the lenght
+    if (i != 13 && i != 15 && i != 16)
+    {
+        printf("INVALID\n");
```

Comment:
```suggestion
        printf("INVALID\n");
        
```

Rule of thumb: Always add a blank line before a return statement.